CMake/utils/kwiver-utils-python.cmake

```diff
@@ -149,10 +149,12 @@ function (kwiver_add_python_module path     modpath    module)
   # in source build this would be kwiver_python_install_path/project_name
   if(SKBUILD)
     set(pyfile_dst "${CMAKE_BINARY_DIR}/${modpath}/${module}.py")
+    set(mod_dist "${CMAKE_BINARY_DIR/${modpath}/")
```

Comment:
Where is this variable used?

Reply:
I don't believe it is. Can you confirm, @johnwparent?

Reply:
This variable is used to pass the final location of the tests module to nose. Used in kwiver-setup-tests-python.cmake line 37.

Reply:
Yes, it absolutely should.

Reply:
This commit was removed in favor of a branch with all of the nose integration: https://github.com/Kitware/kwiver/tree/dev/add_nose. All that should be necessary after that branch is merged into master is a rebase.